docs/backend-api/resources/tracking/tracker/employee.md

```diff
@@ -5,19 +5,39 @@ description: Assigning employee to tracker
 
 # Assigning employee to tracker
 
-## API actions
-
 API base path: `/tracker/employee`
 
+Allows you to assign employee("driver") to a device. Also, read who is on a vehicle now, hardware key and when, where was assigned. 
```

Comment:
- Missed whitespace before `("driver")`.
- It is better not to use the pronoun "you".